pkg/dao/repository_configs_test.go

```diff
@@ -1293,9 +1293,9 @@ func (suite *RepositoryConfigSuite) TestListFilterStatus() {
 	assert.Equal(t, 20, len(response.Data))
```

Comment:
@jlsherrill i think this test failure might be caused by the repoconfig seed options not including references to tasks, so the data returned doesn't include all the repos expected. but i could be wrong. how do you think i should update this?